### ISSUE TITLE
In which method do the system choose hyper-parameters?

### ISSUE THREAD

**Issue by dyj0215** [maxim5/hyper-engine#17]
I am just a little confused.

When i changed the neural network setting with concrete parameters, because i just only want to optimize learning rate, the second training will use a new learning rate to train the neural network.

For example, after the first training, i got a final accuracy=0.8172. 
And then, it will go for a second training with a new learning rate. But the final accuracy maybe be very bad.
I thought, after one training, the second training will use the old results to train this model, and then get a better result. But actually, it may get a worse result.

So i want know, how do this system choose the next learning rate?
Thank you so much.

**Comment by maxim5** [comment]
Your issue is unclear. Please provide the original spec that you used, the output and then changes that you've made.
